rosette/cmaketest.sh

```diff
@@ -18,7 +18,7 @@ cat > "${RESULTS}" < /dev/null
     ulimit -s unlimited
 
     export ESS_SYSDIR=$PWD/rbl/rosette
-    ROSETTE="./build.out/src/rosette --boot rbl/rosette/boot.rbl"
+    ROSETTE="./build.out/src/rosette --boot-dir rbl/rosette"
```

Comment:
Unrelated change.

Reply:
Do you want me to open a new PR for this?

Reply:
Yes please.

Reply:
Removed.